manifests/crd.yaml

```diff
@@ -50,6 +50,9 @@ spec:
     description: The desired replicas number of TiDB cluster
     name: Desire
     type: integer
+  - JSONPath: .metadata.creationTimestamp
```

Comment:
TidbMonitor is missed?

Reply:
no `additionalPrinterColumns` for TidbMonitor, so it will use the default (`NAME` and `AGE`)

```
$ kubectl get tidbmonitor
NAME    AGE
basic   23s
```

Maybe we can set it explicitly?

Reply:
by the way, we can configure some useful columns for TidbMonitor CR. I've created an issue for this. https://github.com/pingcap/tidb-operator/issues/2169